packages/virgo/src/services/range.ts

```diff
@@ -24,6 +24,18 @@ export class VirgoRangeService<TextAttributes extends BaseTextAttributes> {
   onVRangeUpdated = ([newVRange, origin]: VRangeUpdatedProp) => {
     this._vRange = newVRange;
 
+    if (this._vRange === null) {
```

Comment:
Maybe put if after origin type check? This may cause infinite loop with range manager.

Reply:
How it cause?🤔

Reply:
Just want to make sure the `silent` origin works as expect.